package-resources.yml

```diff
@@ -17,7 +17,7 @@ spec:
       - ytt:
           paths:
           - config/package-bundle/config
-          - tmp/output-config.yml
+          - config/relase-config.yml
```

Comment:
`release` spelling is misspelled as `relase`

Reply:
Refactored